packages/inventory/src/EntityTable.js

```diff
@@ -150,11 +153,14 @@ class EntityTable extends React.Component {
     }
 
     createColumns = () => {
-        const { columns, hasItems, rows } = this.props;
+        const { columns, hasItems, rows, expandable: isExpendable } = this.props;
         return columns.map(({ props, transforms, ...oneCell }) => ({
             ...oneCell,
             transforms: [
                 ...this.buildTransforms(props, transforms, hasItems, rows) || []
+            ],
+            cellFormatters: [
+                ...isExpendable ? [ expandable ] : []
```

Comment:
typo i suppose, it should be `isExpandable`

Reply:
That's actually `expandable` from `@patternfly/react-table` package.

Reply:
it says isExp**E**ndable instead of isExp**A**ndable
Just a typo

Reply:
Oh, I thought you were refering to `expandable`, sure thing! I'll change it in the whole file.